guides/common/modules/proc_installing-fapolicyd-on-server.adoc

```diff
@@ -19,12 +19,18 @@ endif::[]
 * Ensure your host has access to the BaseOS repositories of {EL}.
 
 .Procedure
-. Install fapolicyd:
+. For a new installation, install fapolicyd:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # dnf install fapolicyd
 ----
+. For an existing installation, install fapolicyd using {foreman-maintain}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} packages install fapolicyd
```

Comment:
```suggestion
# {project-package-install} fapolicyd
```

upstream+orcharhino: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L187
Satellite: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L95

reason: only Satellite uses "foreman-maintain" to wrap package installation. Please also adjust the introductory sentence.